### DISCUSSION TITLE
Does giscus support pagination?

### DISCUSSION THREAD

**Discussion by Miigon** [giscus/giscus#668] [Q&A] [answered]
It doesn't seem to be mentioned in documentations, and there's no option for it on https://giscus.app/

**Comment by laymonage** [comment, marked as answer]
It's done automatically, you can see that the https://giscus.app itself has pagination in the middle as GitHub does.

<img width="785" alt="image" src="https://user-images.githubusercontent.com/6379424/185564679-d7c80277-68d6-4e74-bff1-e4fd032b58a6.png">
